dataactbroker/handlers/fileHandler.py

```diff
@@ -387,82 +428,82 @@ def getStatus(self):
-                    if missingHeaderString is not None:
+                    missing_header_string = file_results.headers_missing
+                    if missing_header_string is not None:
                         # Split header string into list, excluding empty strings
-                        jobInfo["missing_headers"] = [n.strip() for n in missingHeaderString.split(",") if len(n) > 0]
+                        job_info["missing_headers"] = [n.strip() for n in missing_header_string.split(",") if len(n) > 0]
                     else:
-                        jobInfo["missing_headers"] = []
+                        job_info["missing_headers"] = []
                     # Get string of duplicated headers and parse as a list
-                    duplicatedHeaderString = file_results.headers_duplicated
-                    if duplicatedHeaderString is not None:
+                    duplicated_header_string = file_results.headers_duplicated
+                    if duplicated_header_string is not None:
                         # Split header string into list, excluding empty strings
-                        jobInfo["duplicated_headers"] = [n.strip() for n in duplicatedHeaderString.split(",") if len(n) > 0]
+                        job_info["duplicated_headers"] = [n.strip() for n in duplicated_header_string.split(",") if len(n) > 0]
                     else:
-                        jobInfo["duplicated_headers"] = []
-                    jobInfo["error_type"] = getErrorType(job.job_id)
-                    jobInfo["error_data"] = getErrorMetricsByJobId(
+                        job_info["duplicated_headers"] = []
+                    job_info["error_type"] = getErrorType(job.job_id)
+                    job_info["error_data"] = getErrorMetricsByJobId(
                         job.job_id, job_type=='validation', severity_id=RULE_SEVERITY_DICT['fatal'])
-                    jobInfo["warning_data"] = getErrorMetricsByJobId(
+                    job_info["warning_data"] = getErrorMetricsByJobId(
                         job.job_id, job_type=='validation', severity_id=RULE_SEVERITY_DICT['warning'])
                 # File size and number of rows not dependent on error DB
                 # Get file size
-                jobInfo["file_size"] = self.jobManager.getFileSizeById(job.job_id)
+                job_info["file_size"] = sess.query(Job).filter_by(job_id = job.job_id).one().file_size
                 # Get number of rows in file
-                jobInfo["number_of_rows"] = self.jobManager.getNumberOfRowsById(job.job_id)
+                job_info["number_of_rows"] = sess.query(Job).filter_by(job_id = job.job_id).one().number_of_rows
 
                 try :
-                    jobInfo["file_type"] = self.jobManager.getFileType(job.job_id)
+                    job_info["file_type"] = sess.query(Job).options(joinedload("file_type")).filter_by(job_id = job.job_id).one().file_type.name
```

Comment:
Looks like we've already got a job here -- would it make sense to just walk over to
```python
job.file_type.name
```
? It's actually already preloaded due to `lazy='joined'`